docs/user-guide/deployments.md

```diff
@@ -425,15 +425,27 @@ The `.spec.template` is a [pod template](/docs/user-guide/replication-controller
 the same schema as a [Pod](/docs/user-guide/pods), except it is nested and does not have an
 `apiVersion` or `kind`.
 
+In addition to required fields for a Pod, a pod template in a Deployment must specify appropriate
```

Comment:
We don't. 

Added this in deployment / RC docs:
```
i.e. don't overlap with other controllers
```

Added this in label / selector doc:
```
**Note**: the label selectors of two controllers must not overlap within a namespace, otherwise they will fight with each other. 
```